src/system/documents/actor.ts

```diff
@@ -683,32 +691,30 @@ export class CosmereActor<
         const rollData = foundry.utils.mergeObject(
             {
                 data: data as D20RollData,
-                title: `${flavor}: ${this.name}`,
-                chatMessage: false,
+                title: flavor,
                 defaultAttribute: options.attribute ?? skill.attribute,
+                parts: [`@mod`].concat(options.parts ?? []),
```

Comment:
Shouldn't this fail if `options.parts` is not undefined? The parts property just gets overridden by whatever is in `options.parts` due to the mergeObject call?